src/App.vue

```diff
@@ -13,7 +13,7 @@
   </AppHead>
   <div>
     <PvToast />
-    <NavBar v-if="!navbarBlacklist.includes($route.name)" />
+    <NavBar v-if="!navbarBlacklist.includes($route.name) && isFirekitInitialized" />
```

Comment:
Great! Love it! Can you remove the subscribe patterns in the rest of the components?

Or maybe a better approach is to pull this into a separate PR so that it doesn't hold up your LEVANTE-specific enhancements.